File: Java/Tarjeta.java
package CURSOPOOUBER;
import java.util.Date;
public class Tarjeta extends Payment{
    String franquicia;
    Date fechaVencimiento;
    String codigocvv;
    public Tarjeta(Integer id, String franquicia, Date fechaVencimiento, String codigocvv) {
        super(id);
        this.franquicia = franquicia;
        this.fechaVencimiento= fechaVencimiento;
        this.codigocvv = codigocvv;

        this.printDataPayment();
        

    }
}

File: Python/tarjeta.py
from payment import Payment
from datetime import date
class Tarjeta(payment):
    franquicia = str
    fechaVencimiento = date.today()
    cvv = str

    def __init__(self, id, franquicia, fechaVencimiento, cvv):
        super.__init__(id)
        self.franquicia = franquicia
        self.fechaVencimiento = fechaVencimiento
        self.cvv = cvv
        